libcanard/canard.h

```diff
@@ -10,7 +10,8 @@
 /// It is designed for use in robust deterministic embedded systems equipped with at least 32K ROM and 4..8K RAM.
 /// The codebase follows the MISRA C rules, has 100% test coverage, and is validated by at least two static analyzers.
 /// The library is designed to be compatible with any target platform and instruction set architecture, from 8 to 64
-/// bit, little- and big-endian, RTOS-based or baremetal, etc., as long as there is a standards-compliant C11 compiler.
+/// bit, little- and big-endian, RTOS-based or baremetal, etc., as long as there is a standards-compliant C99/C11
+/// compiler.
```

Comment:
Same thing -- if it was C11 only (or C99 only), it would make sense to be specific here. In my previous review, I suggested that C11 be removed completely, not replaced with C99/C11.

```suggestion
/// bit, little- and big-endian, RTOS-based or baremetal, etc., as long as there is a standards-compliant compiler.
```